vernac/vernacentries.ml

```diff
@@ -421,13 +421,12 @@ let dump_universes_gen prl g s =
     close ();
     Exninfo.iraise reraise
 
-(* [XXX] EGJA: loc unused here *)
 let universe_subgraph ?loc kept univ =
   let open Univ in
   let parse q =
     try Level.make (Nametab.locate_universe q)
     with Not_found ->
-      CErrors.user_err Pp.(str "Undeclared universe " ++ pr_qualid q ++ str".")
+      CErrors.user_err ?loc Pp.(str "Undeclared universe " ++ pr_qualid q ++ str".")
```

Comment:
This looks wrong, the location should be the one of the universe rather than the one of the global command.

Reply:
changed